src/averageAges.js

```diff
@@ -20,6 +20,32 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const ages = [];
+  let averageAge = 0;
+  let eteration = 0;
+  const isMan = human => human.sex === 'm';
```

Comment:
Consider creating helper functions and constants variables before your task functions and reuse them inside calculateWomenAverageAge, calculateMenAverageAge, etc.